setup.py

```diff
@@ -54,7 +54,9 @@ def read_version_py(infname):
     # * foolscap >= 0.12.5 has ConnectionInfo and ReconnectionInfo
     # * foolscap >= 0.12.6 has an i2p.sam_endpoint() that takes kwargs
     # * foolscap 0.13.2 drops i2p support completely
-    "foolscap == 0.13.1",
+    # * foolscap >= 20.4 is necessary for Python 3
+    "foolscap == 0.13.1; python_version < '3.0'",
+    "foolscap >= 20.4.0; python_version > '3.0'",
 
```

Comment:
My understanding is that foolscap drops i2p support, so for i2p to work under Python 2 we need < 20.4.0. Yes?

Reply:
Consensus in IRC seems to be dropping i2p support entirely, in which case yes we can simplify this.

Reply:
P.S. This does get a little weird because it means removing `i2p_requires`, and then is `tor_requires` really necessary? Parsing the comment there, does it mean we now want to depend on `foolscap[tor]`? But then I'm seeing `tor` as an explicit option for our own installation? More research required ... 🧐 

https://github.com/tahoe-lafs/tahoe-lafs/blob/76180d248e8cb3c6eef6d6e4c545af9b8ee86309/setup.py#L137-L141

https://github.com/tahoe-lafs/tahoe-lafs/blob/76180d248e8cb3c6eef6d6e4c545af9b8ee86309/setup.py#L130-L135

https://github.com/tahoe-lafs/tahoe-lafs/blob/76180d248e8cb3c6eef6d6e4c545af9b8ee86309/setup.py#L380-L382

Reply:
I mean, it also means ripping i2p out of the codebase entirely. Right? That's [not nothin'](https://gist.github.com/chadwhitacre/950ebcfa288b61310663e2684911dd08).

Reply:
Decided on Thursday call to keep i2p support around and expect txi2p and foolscap to grow py3 support by the time we _really_ need it.